test/onnx/gen_onnx.py

```diff
@@ -5017,6 +5017,31 @@ def prelu_brcst_test():
     return ([node], [arg0, arg1], [arg_out])
 
 
+@onnx_test()
+def qlinearconv_test():
+    # https://xadupre.github.io/draft/onnx/onnx_doc_folder/onnx__QLinearConv.html
+    x = helper.make_tensor_value_info('X', TensorProto.UINT8, [1, 1, 7, 7])
+    sc_x = helper.make_tensor('1', TensorProto.FLOAT, [], [0.00369204697])
```

Comment:
you can also create test where these are 1-D tensor values instead of scalars or num_elements=1

Reply:
I have created many tests now, and one of them is now using a 1x1x1x1 tensor for Weights. Please see if you need something..  Thanks.

Reply:
It is more for code coverage.  inside `bcast_qdq_instr`, it won't hit 1-D tensor case. I don't see any of the other tests being 1D tensors for scale or z-pt.

Reply:
Thanks. This is actually already covered in the real model resnet50-v1-12-int8.

Reply:
![image](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/assets/139581206/bf88a76b-2706-46e7-a5f0-c59909a0adc6)

Reply:
all scale and z-pt have num_elements =1 from what i see.

Reply:
Is it part of CI ?

Reply:
No, but let me add a new graph and a test as you suggested..

Reply:
Done.